engines/order_management/spec/performance/order_management/subscriptions/proxy_order_syncer_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module OrderManagement
+  module Subscriptions
+    describe ProxyOrderSyncer, performance: true do
```

Comment:
Spec with the :performance tag don't actually get run in CI, right?

I guess these are just being moved across though... :+1:

Reply:
yes, just moving. I thought we could delete these, but they are harmless and can be useful in the future. I was not sure so I left them...